play/manual/shuttermint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+(cd .. && bb init:testchain)
+(cd .. && bb chain) &
```

Comment:
a couple additional steps we'd need to add: 
1. `install bb / go / abigen`
2. `export ROLLING_SHUTTER_BOOTSTRAP_SIGNING_KEY=479968ffa5ee4c84514a477a8f15f3db0413964fd4c20b08a55fed9fed790fad`
3. `export ROLLING_SHUTTER_CHAIN_GENESIS_KEYPER=0x440Dc6F164e9241F04d282215ceF2780cd0B755e`